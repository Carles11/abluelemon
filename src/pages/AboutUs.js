import React from 'react'

export default class AboutUs extends React.Component{
    render(){
        return(
            <div className="aboutUs">
                <div className="aboutUs_title">About Us</div>

                <div className="text_aboutUs aboutUs_bg">
                    <h4> A Blue Lemon Productions finds itself in the early stages of creation and is still in the process of being formed as the organizer of the first <br /> Latin World Music Festival in Dubai (UAE).</h4>
                    
                </div>

            </div>
        )
    }
}