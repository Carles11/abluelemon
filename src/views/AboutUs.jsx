// 

import React from 'react'
import Helmet from 'react-helmet'

const AboutUs = () => 
  <section className="app-view">
    <Helmet 
      title='About Us'
      meta={[
        { name:"description", content: `About Us` },
        { property: "og:title", content: `About Us` }
      ]}
    />
    <h1>About Us</h1>  
  </section>

export default AboutUs