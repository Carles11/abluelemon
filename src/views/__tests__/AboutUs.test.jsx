// 

import React from 'react'
import { shallow } from 'enzyme'
import AboutUs from '../AboutUs'

describe('<AboutUs />', () => {
  it('should render itself', () => {
    const component = shallow(<AboutUs />)
    expect(component).toMatchSnapshot()
  })
})