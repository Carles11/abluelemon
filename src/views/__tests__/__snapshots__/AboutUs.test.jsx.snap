// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AboutUs /> should render itself 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <AboutUs />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <HelmetWrapper
          defer={true}
          encodeSpecialCharacters={true}
          meta={
                    Array [
                              Object {
                                "content": "About Us",
                                "name": "description",
                              },
                              Object {
                                "content": "About Us",
                                "property": "og:title",
                              },
                            ]
          }
          title="About Us"
/>,
        <h1>
          About Us
</h1>,
      ],
      "className": "app-view",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "defer": true,
          "encodeSpecialCharacters": true,
          "meta": Array [
            Object {
              "content": "About Us",
              "name": "description",
            },
            Object {
              "content": "About Us",
              "property": "og:title",
            },
          ],
          "title": "About Us",
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "About Us",
        },
        "ref": null,
        "rendered": "About Us",
        "type": "h1",
      },
    ],
    "type": "section",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <HelmetWrapper
            defer={true}
            encodeSpecialCharacters={true}
            meta={
                        Array [
                                    Object {
                                      "content": "About Us",
                                      "name": "description",
                                    },
                                    Object {
                                      "content": "About Us",
                                      "property": "og:title",
                                    },
                                  ]
            }
            title="About Us"
/>,
          <h1>
            About Us
</h1>,
        ],
        "className": "app-view",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "defer": true,
            "encodeSpecialCharacters": true,
            "meta": Array [
              Object {
                "content": "About Us",
                "name": "description",
              },
              Object {
                "content": "About Us",
                "property": "og:title",
              },
            ],
            "title": "About Us",
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "About Us",
          },
          "ref": null,
          "rendered": "About Us",
          "type": "h1",
        },
      ],
      "type": "section",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
